### PR TITLE
CLDR-18170 Update 46.1 release notes

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-18170

Update 46.1 release notes and make it clearer what is in 46 release vs 46.1

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
fyi @aphillips

File: docs/site/downloads/cldr-46.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: CLDR 46 Release Note
...
+| 46.1 | 2024-12-18 | [v46.1](/index/downloads/cldr-46#461-changes) | n/a | [Charts46.1](https://unicode.org/cldr/charts/46.1/) | [LDML46.1](https://www.unicode.org/reports/tr35/tr35-74/tr35.html) | [Δ46.1](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246.1%22+ORDER+BY+priority+DESC) | [release-46-1](https://github.com/unicode-org/cldr/releases/tag/release-46-1) | [ΔDtd46.1](https://www.unicode.org/cldr/charts/46.1/supplemental/dtd_deltas.html) | [46.1.0](https://github.com/unicode-org/cldr-json/releases/tag/46.1.0)) |
```
Comment (srl295) [inline docs/site/downloads/cldr-46.md:10]:
```suggestion
| 46.1 | 2024-12-18 | [v46.1](/index/downloads/cldr-46#461-changes) | n/a | [Charts46.1](https://unicode.org/cldr/charts/46.1/) | [LDML46.1](https://www.unicode.org/reports/tr35/tr35-74/tr35.html) | [Δ46.1](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246.1%22+ORDER+BY+priority+DESC) | [release-46-1](https://github.com/unicode-org/cldr/releases/tag/release-46-1) | [ΔDtd46.1](https://www.unicode.org/cldr/charts/46.1/supplemental/dtd_deltas.html) | [46.1.0](https://github.com/unicode-org/cldr-json/releases/tag/46.1.0) |
```

one extra paren

**Review by srl295 — DISMISSED** [review]
LGTM there's a stray paren (could fix in a followon)

**Comment by AEApple** [comment]
Might as well fix it now.

**Review by srl295 — APPROVED** [review]

File: docs/site/downloads/cldr-46.md [hunk 2]
```diff
@@ -303,6 +287,36 @@ These matched the ordering of two legacy character encodings. [CLDR-16062][]
...
+## 46.1 Changes
+Version 46.1 is a dot release.
+
+The major change is to bring the Message Format 2.0 specification in Part 9 to Final Candidate status.
+
+Addition standard functions and options are available:
+ - Define REQUIRED and RECOMMENDED functions and options in the function registry.
+ - Number, date/time, and string functions
+ - Add :u options to the :u namespace
+ - Add caledar and numberingSystem options to date/time functions.
```
Comment (stenshamn) [inline docs/site/downloads/cldr-46.md:299]: Typo in "caledar"

File: docs/site/downloads/cldr-46.md [hunk 3]
```diff
@@ -303,6 +287,36 @@ These matched the ordering of two legacy character encodings. [CLDR-16062][]
...
+ - Add hour12 date/time override option
```
Comment (stenshamn) [inline docs/site/downloads/cldr-46.md:300]: Should this be "12-hour"?

File: docs/site/downloads/cldr-46.md [hunk 4]
```diff
@@ -303,6 +287,36 @@ These matched the ordering of two legacy character encodings. [CLDR-16062][]
...
+ - Javascript: TBD
+There is new test data available at: TBD
+
+There are many other spec changes; for a full list, see [Message Format Modifications](https://www.unicode.org/reports/tr35/tr35-74/tr35.html#message-format-modifications)
+The most significant changes to Message Format 2.0 are:
+
+Other changes include
+- More explicit well-formedness and validity constraints for unit of measurement identifiers
+- Addition of emoji annotations that were missing: skintones with emoji facing right
+- Fixes to ja, ko, zh, yue datetimeSkeletons
```
Comment (stenshamn) [inline docs/site/downloads/cldr-46.md:315]: Language codes should be sorted alphabetically.

File: docs/site/downloads/cldr-46.md [hunk 5]
```diff
@@ -303,6 +287,36 @@ These matched the ordering of two legacy character encodings. [CLDR-16062][]
...
+- Improved datetime date time test data
```
Comment (stenshamn) [inline docs/site/downloads/cldr-46.md:316]: "datetime date time"?

File: docs/site/downloads/cldr-46.md [hunk 6]
```diff
@@ -303,6 +287,36 @@ These matched the ordering of two legacy character encodings. [CLDR-16062][]
...
+There are 3 tech preview implementations available
+ - ICU4J: TBD
+ - ICU4C: TBD
+ - Javascript: TBD
+There is new test data available at: TBD
+
+There are many other spec changes; for a full list, see [Message Format Modifications](https://www.unicode.org/reports/tr35/tr35-74/tr35.html#message-format-modifications)
+The most significant changes to Message Format 2.0 are:
+
+Other changes include
```
Comment (stenshamn) [inline docs/site/downloads/cldr-46.md:312]: The bullet list has inconsistent punctuation.

**Pull request merged**